packages/backend/src/config/parseConfig.ts

```diff
@@ -406,7 +405,12 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
                 'https://analytics.lightdash.com',
         },
         sentry: {
-            dsn: process.env.SENTRY_DSN || '',
+            backend: {
+                dsn: process.env.SENTRY_BE_DSN || '',
+            },
+            frontend: {
+                dsn: process.env.SENTRY_FE_DSN || '',
+            },
```

Comment:
Should we also change this here to: @ZeRego ?

```suggestion
            backend: {
                dsn: process.env.SENTRY_BE_DSN || process.env.SENTRY_DSN || '',
            },
            frontend: {
                dsn: process.env.SENTRY_FE_DSN || process.env.SENTRY_DSN || '',
            },
```

Reply:
I think that would be a nice backwards compatibility touch. 👍

Reply:
done: https://github.com/lightdash/lightdash/pull/9910/commits/2e3bc7e2000ebe63ff1f5d624cc8778a330e3622